test/e2e/framework.go

```diff
@@ -2778,6 +2778,34 @@ func (data *TestData) GetTransportInterface() (string, error) {
 	return "", fmt.Errorf("no interface was assigned with Node IP %s", nodeIP)
 }
 
+func (data *TestData) GetTransportInterfaceIPNet(nodeName string, nodeIP string) (*net.IPNet, error) {
```

Comment:
can we clean up the code and unify with `GetTransportInterface`?

I think we could have:
```go
// GetTransportInterfaceForNode returns the transport interface name for a given Node, along with the IP prefixes assigned to it
// TODO: maybe it would be better to take the nodeName as parameter?
func (data *TestData) GetTransportInterfaceForNode(nodeIdx int) (string, []*net.IPNet, error) {

}

// GetTransportInterfaceName returns the transport interface name for cluster Nodes, assuming all Nodes have the same one.
func (data *TestData) GetTransportInterfaceName() (string, error) {
        name, _, err := data.GetTransportInterfaceForNode(0)
        return name, err
}
```